tools/setup/emoji/build_emoji

```diff
@@ -144,7 +144,7 @@ def code_point_to_file_name_map(ttx):
     """
     result = {}  # type: Dict[int, Union[Text, bytes]]
     xml = ET.parse(ttx)
-    for elem in xml.find("*cmap_format_12"):
+    for elem in xml.iterfind("*cmap_format_12"):
```

Comment:
I think what's going on here is that:
* `xml.find(...)` gets us one element, and then
* we iterate through *that* element. Which gives us its child elements.

This isn't super clear, and I'd be glad to see an edit to make it clearer that that's what's happening -- even just adding something like `cmap = xml.find("*cmap_format_12")` and then `for elem in cmap:` is probably an improvement.